.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: execute garoon-plugin-uploader
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+          cache: 'yarn'
+      - run: |
+          yarn install --immutable
```

Comment:
Sorry, `--immutable` is not support on yarn 1.x. Use `--frozen-lockfile` instead.